Makefile

```
@@ -2,6 +2,8 @@ PROJECTPATH = $(dir $(realpath $(firstword $(MAKEFILE_LIST))))
 export GOPATH := $(PROJECTPATH)
 export GOCACHE := $(GOPATH)/.gocache
 export SRCDIR := $(PROJECTPATH)src/hockeypuck
+VERSION ?= $(shell git describe --tags 2>/dev/null)
+TIMESTAMP = $(shell date -Iseconds)
```

Suggestion: normalize the timezone to UTC,

```suggestion
TIMESTAMP = $(shell date -Iseconds -u)
```

---

I can't believe I missed that; I'm normally the UTC police in $WORK. :-)